src/CryptoNoteCore/Core.cpp

```diff
@@ -2099,9 +2099,11 @@ void Core::importBlocksFromStorage() {
     CachedBlock cachedBlock(blockTemplate);
 
     if (blockTemplate.previousBlockHash != previousBlockHash) {
-      logger(Logging::ERROR) << "Corrupted blockchain. Block with index " << i << " and hash " << cachedBlock.getBlockHash()
-                             << " has previous block hash " << blockTemplate.previousBlockHash << ", but parent has hash " << previousBlockHash
-                             << ". Resynchronize your daemon please.";
+      logger(Logging::ERROR) << "Local blockchain corruption detected. " << std::endl
+                             << "Block with index " << i << " and hash " << cachedBlock.getBlockHash()
+                             << " has previous block hash " << blockTemplate.previousBlockHash << ", but parent has hash " << previousBlockHash << "." << std::endl
+                             << "Please try to repair this issue by starting the node with the option: --rewind " << i << std::endl
```

Comment:
```suggestion
                             << "Please try to repair this issue by starting the node with the option: --resync-from-height " << i << std::endl
```

Reply:
I suspected there was another reference in the description to it somewhere! ty!